detox/src/logger/utils/streams/DetoxJSONLParser.js

```diff
@@ -0,0 +1,31 @@
+const JsonlParser = require('stream-json/jsonl/Parser');
+
+const { through } = require('./transformers');
+
+class DetoxJSONLParser {
+  constructor(log) {
+    this._log = log;
+  }
+
+  createTransformer() {
+    const log = this._log;
+    const readable = through();
+    const writable = JsonlParser.make({ checkErrors: true })
+      .on('error', (err) => {
+        /* istanbul ignore else */
+        if (err instanceof SyntaxError) {
+          log.debug({ err });
+          readable.end();
```

Comment:
I'm not sure what `SyntaxError` stands for in this context, but assuming (guessing) that it is about a specific json line then wouldn't it be better to skip (and provide some other "wrapper" log), instead? Something along the lines of:
`<Failed to parse line: ${err}>`

Reply:
It stands for the unfinished JSON object on the last line of JSONL file.

Reply:
The idea with "Failed to parse line" seems nice. 👍 TODO